src/pkg/utils/utils.go

```diff
@@ -708,23 +710,27 @@ func ResolveImageName(distro, image, release string) (string, string, error) {
 	logrus.Debugf("Release: '%s'", release)
 
 	if distro == "" {
-		distro = distroDefault
+		distro = viper.GetString("general.distro")
 	}
 
 	if distro != distroDefault && release == "" {
```

Comment:
This condition needs to be updated.

The idea is that if we have a non-default `distro` (ie., one that doesn't match `/etc/os-release`), then we need the user to specific a `release` either through the command line or the configuration file. Currently, this isn't checking the presence of a release in the configuration file.